packages/helpers/README.md

```diff
@@ -24,6 +24,39 @@ Used by the Mail class to compose `Personalization` objects.
 ## Email address
 `Helper` class to represent an email address with name/email. Used by both the `Mail` and `Personalization` classes to deal with email addresses of various formats.
 
+### Usage
```

Comment:
Usage of this class isn't really required. It's primarily used internally. Recommend looking into the TypeScript definitions if you're wanting some compile-time checking.

Either way, if you still want to add these docs, I think it best to put them next to the other usages like here: https://github.com/sendgrid/sendgrid-nodejs/blob/master/docs/use-cases/cc-bcc-reply-to.md

Reply:
Hey @childish-sambino 

Just to clarify, are you saying i should create a use case markdown for the helper library?

Reply:
Yes, well, the existing markdown files should be extended and/or supplemented with this information. But all the usage docs be under that folder.